src/include/autodetectcompilers.h

```diff
@@ -17,11 +18,44 @@ class AutoDetectCompilers : public wxScrollingDialog
 		AutoDetectCompilers(wxWindow* parent);
 		~AutoDetectCompilers() override;
 
-		void OnDefaultClick(wxCommandEvent& event);
+        bool closeCheckOkay();
+        void OnClose(wxCloseEvent& event);
```

Comment:
The indent is wrong here...

Reply:
Can you check your editor to ensure that tabs are set to show as 4 spaces.  The lines I have added have 8 spaces and the lines above and below have 2 x tab characters.

Most lines use spaces.

Reply:
I don't use an editor other than the diff view of github.